blackbox/docs/config/logging.rst

```diff
@@ -99,3 +99,42 @@ Settings<sys-cluster-settings>`).
 
 .. _Log4j: https://logging.apache.org/log4j/1.2/
 .. _`hierarchical named loggers`: https://logging.apache.org/log4j/1.2/manual.html
+
+.. _conf-logging-gc:
+
+Garbage Collection Logging
+==========================
+
+Addition to the regular application logging, CrateDB also logs garbage
+collection times of the Java Virtual Machine using the built-in GC logging of
```

Comment:
can we refer to it as "garbage collection" in full in every instance?

Reply:
sure